views/partials/compromised_accounts.hbs

```diff
@@ -3,7 +3,7 @@
   {{#each foundBreaches }}
     <div class="account">
       <div class="image-wrap">
-        <img alt="{{ name }}" src="img/{{ name }}.png">
+        <img alt="{{ name }}" src="img/logos/{{ name }}.png">
```

Comment:
I can file this as a separate issue if we don't want to block on this right meow, but...

This won't work for roughly 50% of our logos since we use a mix of SVG and PNG files.

@groovecoder I noticed in the HIBP /api/v2/breaches endpoint, that they expose a "LogoType" attribute of `"png"` or `"svg"`. Is that available in our breaches table DB meta somewhere?

```js
LogoType: "png"
```

Reply:
Yup, I think that png/svg data should be in our DB meta field.